docs/configuration-reference/components/flatcar-linux-update-operator.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Flatcar Container Linux update operator configuration reference for Lokomotive
+title: Flatcar Container Linux Update Operator configuration reference for Lokomotive
+linkTitle: Flatcar Container Linux Update Operator
```

Comment:
Hmm, same here. It should be `Flatcar Linux Update Operator`.

Reply:
Sure, done.